crates/db_views/src/site_view.rs

```diff
@@ -25,12 +30,14 @@ impl SiteView {
       ))
       .first::<(Site, LocalSite, LocalSiteRateLimit, SiteAggregates)>(conn)
       .await?;
+    let taglines = Tagline::get_all(pool, local_site.id).await.ok();
```

Comment:
Okay, I read this wrong. SiteView is only for the joinable site columns, so this is wrong. It should not be doing multiple fetches here. 

taglines needs to come back with GetSiteResponse, not the SiteView.

Reply:
That means lemmy-js-client also needs fixed.

Reply:
Another thing: I was wrong here about `Result` being an error if there are no results returned; diesel will only have an Error if you use `.first`, not `get_results`. `get_results` unfortunately will still return an empty vector.

So do this:

```rust
let taglines_res = Tagline::get_all(pool, local_site.id).await?;
let taglines = !taglines_res.is_empty()).then_some(taglines_res);
```

Reply:
Okay updated all repos, new PR for js-client is up